src/desktop/main.ts

```diff
@@ -205,39 +206,19 @@ expressApp.use("/dialog/content/suggestLocation", async (req, res) => {
 });
 
 expressApp.use("/dialog/login", async (req, res) => {
-  openLoginWindow();
-  res.send({ message: "SUCCESS", responseCode: "OK" })
+  const options = {"request":{"type":"desktopConfig","action":"get","subType":"login"}};
+  HTTPService.post(`${appBaseUrl}/api/data/v1/form/read`, options).toPromise().then(async response => {
+    let loginFormConfig = _.find(response.data.result.form.data.fields, (field) => field.context == 'login');
+    const loginSessionProvider = new LoginSessionProvider(win, loginFormConfig, appBaseUrl);
+    loginSessionProvider.childLoginWindow().then((rs) =>{
+      res.send({ message: "SUCCESS", responseCode: "OK" })
+    })
+  }, err => {
+    res.send({ message: "Login failed", responseCode: "OK" })
```

Comment:
in err case responseCode should not be **OK**

Reply:
done